### ISSUE TITLE
Add Tags in _Split Transaction_ screen 

### ISSUE THREAD

**Issue by wolfsolver** [moneymanagerex/android-money-manager-ex#1993]
The _Split Transaction_ screen still only shows the three standard fields - _Category_, _Amount_ and _Notes_, like this:  

<kbd>

![AMMEx - Split Trans](https://github.com/user-attachments/assets/0dfb0d49-1ed1-4010-a22f-b8ac5118a808)

</kbd>
 

Please add the new _Tags_ field to this Split Transaction screen so that each item in a split transaction can be assigned one or more different tags, like this:

<kbd>

![AMMEx - Split Trans](https://github.com/user-attachments/assets/a9014c9d-0de1-450b-8483-b666e72f5314)

</kbd>

_Originally posted by @luisdev in https://github.com/moneymanagerex/android-money-manager-ex/issues/735#issuecomment-2558462065_

**Comment by MisterEco** [comment]
It will be nice to have such option!

**Comment by froux07** [comment]
Yes, very good idea. Because the search by tag, does not retrieve split transactions
:+1:
